CMakeLists.txt

```diff
@@ -284,7 +284,11 @@ add_executable(UnitTestsRunner
     "${service_output_dir}/nifake/nifake_service.cpp"
     "${custom_dir}/nifake_service.custom.cpp"
     "${proto_srcs_dir}/nifake.pb.cc"
-    "${proto_srcs_dir}/nifake.grpc.pb.cc")
+    "${proto_srcs_dir}/nifake.grpc.pb.cc"
+    "${service_output_dir}/nifake_extension/nifake_extension_service.cpp"
+    "${custom_dir}/nifake_extension_service.custom.cpp"
+    "${proto_srcs_dir}/nifake_extension.pb.cc"
+    "${proto_srcs_dir}/nifake_extension.grpc.pb.cc")
```

Comment:
I've generally tried to keep these organized such that the files in a given directory are grouped together. That being said, I could see treating that differently here such that the fake extensions are grouped.